common/test/acceptance/pages/lms/problem.py

```diff
@@ -84,6 +84,13 @@ def click_hint(self):
         self.q(css='div.problem button.hint-button').click()
         self.wait_for_ajax()
 
+    def click_choice(self, choice_value):
```

Comment:
If this operation is for specifically clicking a 'radio' choice then let's state that in the operation name -- if it applies to more than just radio choices we should modify the docstring